packages/edit-site/src/components/block-editor/index.js

```diff
@@ -201,8 +184,10 @@ export default function BlockEditor( { setIsInserterOpen } ) {
 	// Optimise for dead code elimination.
 	// See https://github.com/WordPress/gutenberg/blob/trunk/docs/how-to-guides/feature-flags.md#dead-code-elimination.
 	let MaybeNavMenuSidebarToggle = Fragment;
+	const isOffCanvasNavigationEditorEnabled =
+		window?.__experimentalEnableOffCanvasNavigationEditor === true;
 
-	if ( process.env.IS_GUTENBERG_PLUGIN ) {
+	if ( isOffCanvasNavigationEditorEnabled ) {
```

Comment:
Because the code changed from `process.env.IS_GUTENBERG_PLUGIN` to `isOffCanvasNavigationEditorEnabled`, the comment above about dead code elimination no longer applies and could be removed.

The same comment applies to the other file which has the same comment.